rating.jenkins.io.tf

```
@@ -16,13 +16,10 @@ resource "postgresql_role" "rating" {
 # This (sensitive) output is meant to be encrypted into the production secret system, to be provided as a secret to the ratings.jenkins.io application
 output "rating_dbconfig" {
   sensitive   = true
-  description = "file /config/dbconfig.php"
+  description = "dbconfig values"
   value       = <<-EOT
-    <?php
-      $dbuser='${local.public_pgsql_admin_login}';
-      $dbpass='${random_password.pgsql_rating_user_password.result}';
-      $dbname='rating';
-      $dbserver='${azurerm_postgresql_flexible_server.public.fqdn}';
-      $dbport='';
+      username=${local.public_pgsql_admin_login}
+      password=${random_password.pgsql_rating_user_password.result}
+      server=${azurerm_postgresql_flexible_server.public.fqdn}
```

```suggestion
database:
  username: "${local.public_pgsql_admin_login}"
  password: "${random_password.pgsql_rating_user_password.result}"
  server: "${azurerm_postgresql_flexible_server.public.fqdn}"
  name: "${postgresql_database.rating.name}"
```

- As per https://github.com/jenkins-infra/helm-charts/blob/main/charts/rating/values.yaml#L73-L78, you want to map the YAMl structure (so you can copy and paste directly the yaml into the SOPS secret
- You also need the name of the database. It has to be retrieved dynamically